README.md

```diff
@@ -9,51 +9,37 @@ In order to deploy the Marketplace Operator, you must:
 
-- `endpoint` is typically set to `https:/quay.io/cnr` if you are using Quay's app-registry.
+- `updateStrategy` is used to determine the frequency at which the source image is polled for `grpc` type `CatalogSources`. The update takes some time to complete, so the `interval` should not be too short. An interval of `10m` - `15m` should be sufficient for this.
 
-- `registryNamespace` is the name of your app-registry namespace.
+- `secrets` are a list of secrets used to access contents of the catalog. These are tried for every catalog entry, so this list should be kept short.
 
-- `displayName` and `publisher` are optional and only needed for UI purposes.
+- `sourceType` specifies the data source type that the catalog source references. Supported `sourceTypes` include `"grpc"` and `"configmap"`. The recommended source type is `"grpc"`.
 
-Please see [here][community-operators] for an example `OperatorSource`.
+- `image` is the registry image that is queried for `grpc` type `CatalogSources`.
 
-If you want an `OperatorSource` to work with private app-registry repositories, please take a look at the [Private Repo Authentication](docs/how-to-authenticate-private-repositories.md) documentation.
+- `address` is specified as \<host or ip>:\<port> and can be used to connect to a pre-existing registry for `grpc` type `CatalogSources`. This field is ignored if the `image` field is non-empty.
 
-On adding an `OperatorSource` to an OKD cluster, operators will be visible in the [OperatorHub UI](https://github.com/openshift/console/tree/master/frontend/public/components/operator-hub) in the OKD console. There is no equivalent UI in the Kubernetes console.
+- `configMap` is used in `configmap` type `CatalogSources` to refer to the `ConfigMap` that backs the registry.
 
-The creation of an `OperatorSource` results in the creation of an OLM `CatalogSource` in the same namespace the marketplace operator is running in. This `CatalogSource` will be populated with operators from the `OperatorSource` ready to be managed by OLM.
+- `displayName`, `description`, `icon` and `publisher` are optional and only needed for UI purposes.
 
-#### CatalogSourceConfig
+Please see [here][community-operators] for an example `CatalogSource`.
 
-`CatalogSourceConfig` is used to create OLM `CatalogSources` consisting of operators from one `OperatorSource` so that these operators can then be managed by OLM.
+If you want an `CatalogSource` to work with private registry repositories, please take a look at the [Private Repo Authentication](docs/how-to-authenticate-private-repositories.md) documentation.
```

Comment:
This line has to be removed. We had a way to authenticate private repos using OperatorSources, but currently there's no way to authenticate using CatalogSources, unless the node itself is authenticated (which entails different steps)